packages/react-ui-ag/src/Carousels/ListingCarousel.js

```diff
@@ -30,7 +34,18 @@ export default class ListingCarousel extends PureComponent {
 
   constructor(props) {
     super(props)
-    this.state = { selectedIndex: this.props.selectedIndex }
+    this.state = {
+      selectedIndex: this.props.selectedIndex,
+      activelySliding: false,
+    }
```

Comment:
I don't see where you are using or setting `this.state.activelySliding`?

Reply:
Fixed